gotext.go

```diff
@@ -96,31 +105,45 @@ func GetDomain() string {
 func SetDomain(dom string) {
 	globalConfig.Lock()
 	globalConfig.domain = dom
-	if globalConfig.storage != nil {
-		globalConfig.storage.SetDomain(dom)
+	if globalConfig.locales != nil {
+		for _, locale := range globalConfig.locales {
+			locale.SetDomain(dom)
+		}
 	}
 	globalConfig.Unlock()
 
-	loadStorage(true)
+	loadLocales(true)
 }
 
-// GetLanguage is the language getter for the package configuration
+// GetLanguage returns the language gotext will translate into.
+// If multiple languages have been supplied, the first one will be returned.
 func GetLanguage() string {
-	globalConfig.RLock()
-	lang := globalConfig.language
-	globalConfig.RUnlock()
+	return GetLanguages()[0]
```

Comment:
Nitpicking ; if one calls `SetLocales()` with an empty slice, this will panic.
Calling `SetLocales()` with an empty slice could default back to the `globalConfig` language, or checks could be made here ?

Reply:
Do note that calling `SetLocales` with an empty slice could be desired when writing consumer tests, to sort of reset the state of things to defaults.